### DISCUSSION TITLE
kong start by docker but command 'kong reload' in container is not work

### DISCUSSION THREAD

**Discussion by Canon4G** [Kong/kong#9250] [Help] [answered]
i'm code api-plugin in kong,
but after i change my plugin or create new plugin , i user this command 'kong reload' in container , my change not work,
just docker stop the container and start it to make my plugin work,
it wastes lots of time
what can i do, help!!!

**Comment by kikito** [comment, marked as answer]
This is a result of how Docker and the Docker-kong image works.

I suggest you give Pongo a try instead: https://github.com/kong/kong-pongo

It is a simpler solution tailored for creating custom Plugins. It should be a bit faster to test than your current method.
